Userland/Libraries/LibWeb/Painting/SVGGeometryPaintable.cpp

```diff
@@ -41,58 +41,21 @@ void SVGGeometryPaintable::paint(PaintContext& context, PaintPhase phase) const
     Gfx::AntiAliasingPainter painter { context.painter() };
     auto& svg_context = context.svg_context();
 
-    auto offset = svg_context.svg_element_position();
-    painter.translate(offset);
-
-    auto const* svg_element = geometry_element.first_ancestor_of_type<SVG::SVGSVGElement>();
-    auto maybe_view_box = svg_element->view_box();
+    auto svg_position = svg_context.svg_element_position() * context.device_pixels_per_css_pixel();
+    auto scaling = layout_box().viewbox_scaling() * context.device_pixels_per_css_pixel();
```

Comment:
Instead of doing this here, can we set it up so that the `PaintContext` stores this transform (as part of its `SVGContext` object)? Then we can make all SVG code "viewbox accurate" by simply painting through the current context transform.